src/pfe/file-watcher/server/src/projects/projectUtil.ts

```diff
@@ -967,6 +986,25 @@ export async function isContainerActive(projectID: string, handler: any): Promis
     }
 }
 
+/**
+ * @function
+ * @description Set the ODO app name and save it to the project info file.
+ *
+ * @param projectInfo <Required | ProjectInfo> - The metadata information for a project.
+ *
+ * @returns Promsie<void>
+ */
+export async function setOdoAppName(projectInfo: ProjectInfo): Promise<void> {
```

Comment:
Not sure, if I want to know call this getOdoAppName now; because the we are actually getting the name using the odo command.. Maybe call it getAndSaveOdoAppName?